<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>returnDialog</class>
 <widget class="QDialog" name="returnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#returnDialog{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #320453, stop:1 #802CBD);

}
QGroupBox#groupBox{
	background-color:#7F00FF;
	border-radius: 30px;
}
QPushButton{
	border-radius: 25px;
	background-color: black;
	font-size: 18px;
	font-weight: bold;
}
QPushButton#confirmButton{
background-color: #1DB500;
	
}
QPushButton#confirmButton:hover{
background-color:#308420;
}
QPushButton#cancelButton{
	background-color:#FF4C4C;
}
QPushButton#cancelButton:hover{
	bakcground-color: #8C1717;	
}

</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>381</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Return Book</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Title : </string>
    </property>
   </widget>
   <widget class="QLabel" name="AuthorLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Author : </string>
    </property>
   </widget>
   <widget class="QLabel" name="yearLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Year : </string>
    </property>
   </widget>
   <widget class="QLabel" name="isbnLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ISBN : </string>
    </property>
   </widget>
   <widget class="QLabel" name="returnDateLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Return Date : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ENSIA Library Management System</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
